tests/plots/test_scatter.py

```diff
@@ -0,0 +1,52 @@
+import matplotlib.pyplot as plt
+import pytest
+
+import shap
+
+
+@pytest.mark.mpl_image_compare
+def test_scatter_single(explainer):
+    explanation = explainer(explainer.data)
+    shap.plots.scatter(explanation[:, "Age"], show=False)
+    fig = plt.gcf()
+    plt.tight_layout()
+    return fig
```

Comment:
Can you also add a test for the multiple columns variant of scatter plot?

Reply:
Good suggestion, done.

One bit that remains uncovered by tests is plotting interaction values, in this block:
https://github.com/shap/shap/blob/a82f014a5cbc08f9ae3ece6ad2b08a603dc2cb48/shap/plots/_scatter.py#L250-L251

 I couldn't work out the correct input data to invoke this functionality , so if you know how to do this that would make for a useful follow-up PR.

Reply:
> I couldn't work out the correct input data to invoke this functionality

I took a glance, neither could I. Isn't it dead code though, since `ind=0` is set at the beginning, then `convert_name(ind,...)` will return 0 since it's not a string. It looks awfully like dead code, but let's leave this for a future PR.

---

ps: I just noticed in our docstrings, we only mentioned single column. Let's edit this as well while we're at it? We technically support multiple columns + the behaviour is tested now.

![image](https://github.com/user-attachments/assets/920fc3b7-5dff-409a-a698-5f53d08101c7)

Reply:
Good shout, done.